saagieapi/env_vars/env_vars.py

```diff
@@ -415,3 +416,74 @@ def export(self, project_id, output_folder: str, error_folder: Optional[str] = "
             write_error(error_folder, "env_vars", project_id)
             result = False
         return result
+
+    def import_from_json(self, json_file: str, project_id: str = None) -> bool:
+        """Import environment variables from JSON format
+        Parameters
+        ----------
+        json_file : str
+            Path to the JSON file that contains env var information
+        project_id : str, optional
+            Project ID
+        Returns
+        -------
+        bool
+            True if environment variables are imported False otherwise
+        """
+        result = True
+
+        try:
+            with open(json_file, "r") as file:
+                env_var_info = json.load(file)
+        except Exception as e:
+            logging.warning("Cannot open the JSON file %s", json_file)
+            logging.error("Something went wrong %s", e)
+            return False
+
+        try:
+            env_var_id = env_var_info["id"]
```

Comment:
env_var_id not used

Reply:
unused variable deleted.
I also checked import_from_json on job and pipeline. I found an unused variable on job and deleted it.